08_voting/index.tex

```diff
@@ -1 +1,101 @@
 \chapter{Voting}
+\section{Voting Scenarios}
+\label{sec:VotingScenarios}
+
+Within the context of the coursework, robust voting systems are designed for deciding the outcome of collective actions (e.g. elections for legislative, executive and judicial branches in IIGO).
+When voting is called in IIGO, the President, Speaker, Judge and the islands in the archipelago (i.e. the \emph{Voters}, assuming that no diplomatic sanction is in place) will be empowered to proceed with the following series of actions:
+\begin{itemize}
+    \item The President will be eligible to select a rule for the archipelago to vote for.
+    \item The Speaker will be eligible to take charge of enabling the voting procedure by receiving, counting ballots and ultimately, declaring the winner.
+    \item The islands in the archipelago (i.e. Voters) will be eligible to vote by submitting their ballots\footnote{This is based on the assumption that no diplomatic sanctions are in place for any islands.}.
+\end{itemize}
+
+In an IIGO session, the voting action is needed for deciding the next President, Speaker, Judge and also for deciding whether or not to adapt a new rule as well as to disable an active one.
+
+\textbf{Elections of IIGO roles:} As per the IIGO specification, the elections of roles will be conducted according to the accountability cycle (see Section XX): The Judge can initiate the transfer-of-power for the Speaker; the President can initiate the transfer-of-power for the Judge; the Speaker can initiate the transfer-of-power for the President. To prevent multiple-role enactment of the same island, the island that has been selected for a role will be removed from the candidate list for other roles in the current turn\footnote{Note that this assumes that the following rule is in place: The President, Speaker and Judge are all different islands.}. Depending on the sequence of the elections called, the candidates for three roles will consist of maximally six, five, and four islands accordingly. The ballots from the Voters will be in the format of preference lists which rank all candidates in the desired order.
```

Comment:
Could you change Section XX to correctly reference the appropriate chapter/section

Reply:
I wrote Section XX.. the section  that @walkeronwater  is not present  now -- wil come later

Reply:
Will do this later when section accountability cycle is available